test/benchmark/README.md

```diff
@@ -57,7 +96,7 @@ python report.py
 ```
 The final benchmark report will be available in markdown format as `report.md` in the `serve/` folder. 
 
-**Example report for vgg16 model**
+**Example report for vgg11 model**
```

Comment:
What is the reason for switching to vgg11 here?

Reply:
The results quoted below are vgg11 results, this 'vgg16' was just a typo that was corrected.